### PR TITLE
New release v1.2.0

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by s1na** [comment]
On second thought, seems to me that doing a release with only place-holder values for the HF can be somewhat confusing. I wonder if we can for example use `ethereumjs-common`'s master branch (with the merged #51) for testing and release 1.2.0 with full Istanbul release? Or do you see a strong case for doing the release now and working towards a complete support in multiple minor releases?

**Comment by holgerd77** [comment]
@s1na I think we'll likely run into caching problems if we just use `master`. I would also like to release draft `Istanbul` libraries early on for experimentation, since the final Istanbul version is still months away, and in the official library releases I wouldn't want to use `master` Common version as dependency.

So I still have a strong tendency to do these as official releases. The Common library is just used by us anyhow.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+**DRAFT Istanbul Hardfork Support**
+
+Draft support for the upcoming `Istanbul` hardfork planned for October 2019,
+use `istanbul` as constructor `hardfork` parameter to activate. Parameters
+relevant to new EIPs accepted for the HF will be added along subsequent `1.2.x`
+releases, the finalized HF version will be released along a subsequent `1.x.0`
+release (likely `1.3.0`).
+
+See new `hardforks/istanbule.json` file as well as PR
```
Comment (s1na) [inline CHANGELOG.md]: Nitpick: should be `hardforks/istanbul.json` :)

Comment (holgerd77) [reply CHANGELOG.md]: Oh, definitely no nitpick, updated. 😄

**Pull request merged**